docs/guides/basics/data-types.md

```diff
@@ -315,29 +315,32 @@ The [`Reference`][Reference] value hint can be used to reference a row in a sync
 
 | Column type   | Supported | Value type           | Value hint        |
-|---------------|-----------|----------------------|-------------------|
-| Text          | ✅ Yes     | `String`             |                   |
-| Select list   | ❌ No      |                      |                   |
-| Number        | ✅ Yes     | `Number`             |                   |
-| Percent       | ✅ Yes     | `Number`             | `Percent`         |
-| Currency      | ✅ Yes     | `Number`             | `Currency`        |
-| Slider        | ✅ Yes     | `Number`             | `Slider`          |
-| Scale         | ✅ Yes     | `Number`             | `Scale`           |
-| Date          | ✅ Yes     | `String` or `Number` | `Date`            |
-| Time          | ✅ Yes     | `String` or `Number` | `Time`            |
-| Date and time | ✅ Yes     | `String` or `Number` | `DateTime`        |
-| Duration      | ✅ Yes     | `String`             | `Duration`        |
-| Checkbox      | ✅ Yes     | `Boolean`            |                   |
-| People        | ✅ Yes     | `Object`             | `Person`          |
-| Reaction      | ❌ No      |                      |                   |
-| Button        | ❌ No[^1]  |                      |                   |
-| Image         | ✅ Yes     | `String`             | `ImageAttachment` |
-| Image URL     | ✅ Yes     | `String`             | `ImageReference`  |
-| File          | ✅ Yes     | `String`             | `Attachment`      |
-| Lookup        | ✅ Yes     | `Object`             | `Reference`       |
+| ------------- | --------- | -------------------- | ----------------- |
+| Text          | ✅ Yes    | `String`             |                   |
+| Link          | ✅ Yes    | `String`             | `Url`             |
+| Canvas        | ❌ No     |                      |                   |
+| Select list   | ❌ No     |                      |                   |
+| Number        | ✅ Yes    | `Number`             |                   |
+| Percent       | ✅ Yes    | `Number`             | `Percent`         |
+| Currency      | ✅ Yes    | `Number`             | `Currency`        |
+| Slider        | ✅ Yes    | `Number`             | `Slider`          |
+| Scale         | ✅ Yes    | `Number`             | `Scale`           |
+| Date          | ✅ Yes    | `String` or `Number` | `Date`            |
+| Time          | ✅ Yes    | `String` or `Number` | `Time`            |
+| Date and time | ✅ Yes    | `String` or `Number` | `DateTime`        |
+| Duration      | ✅ Yes    | `String`             | `Duration`        |
+| Checkbox      | ✅ Yes    | `Boolean`            |                   |
+| People        | ✅ Yes    | `Object`             | `Person`          |
+| Email         | ❌ No     |                      |                   |
```

Comment:
just curious are we planning on supporting this? And is the difference between this section and the "parameters" section that this section is talking about what types we support as result types?

Reply:
I don't know if anyone is working on it, but ideally we would support it. This section is what value hints you can apply to your formula or schema property to get the equivalent column type in Coda. The parameters version is about what parameter type we recommend you use to pass that value to your formula.

Reply:
Ahhh gotcha thank you!

Reply:
ahh gotcha thank you!

Reply:
Oh Mike has PRs out right now to add support for this.

Reply:
Sweet! Will update this table once that PR is in.